### PR TITLE
A few more examples

### PR DESCRIPTION
**Pull request by tmcgilchrist**
http://docs.datomic.com/query.html#outline-container-5-17-1

### CODE REVIEW

File: doc/icicle-concrete.md [hunk 1]
```diff
@@ -498,6 +491,12 @@ if (feat != feat_end) {
 return days;
 ```
 
+Datalog
+```
+;; Basically a the min of the max 5
```
Comment (amosr) [inline doc/icicle-concrete.md]: ? why 5?

**Comment by amosr** [comment]
:+1: more examples always more good

File: doc/icicle-concrete.md (hunk 1 above)
Comment (tmcgilchrist) [inline doc/icicle-concrete.md]: Might have misunderstood the example, isn't it asking for the nth day since the Earliest, where n is 5 here?

Comment (amosr) [inline doc/icicle-concrete.md]: oh right, no - it's asking how many days ago the earliest entry is

**Pull request merged**